src/Components/ProductDetails/ProductQuantity/QuantityCounter/index.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useState } from "react";
+import Button from "@mui/material/Button";
+import AddIcon from "@mui/icons-material/Add";
+import RemoveIcon from "@mui/icons-material/Remove";
+import style from "./index.module.css";
+import Box from "@mui/material/Box";
+
+const QuantityCounter = () => {
+  const [counter, setCounter] = useState(0);
+  const { textStyle } = style;
+
+  const handleAdd = () => {
+    setCounter(counter + 1);
+  };
```

Comment:
This arrow function consists of one line, so let's have it in one line without the parentheses.